src/Symfony/Component/Emoji/Resources/bin/Makefile

```diff
@@ -4,7 +4,7 @@
 update: ## Update sources
 	@composer update
 	@curl https://api.github.com/emojis > vendor/github-emojis.json
-	@curl https://gitlab.com/gitlab-org/gitlab/-/raw/master/fixtures/emojis/index.json > vendor/gitlab-emojis.json
+	@curl https://gitlab.com/gitlab-org/gitlab/-/raw/master/fixtures/emojis/digests.json > vendor/gitlab-emojis.json
```

Comment:
The file has been removed https://gitlab.com/gitlab-org/gitlab/-/merge_requests/166790/diffs#diff-content-d18604294edc1ef17fc6d690885df5ebb7f530e9